### PR TITLE
Feature/post only #11

### PR DESCRIPTION
**Pull request by ruud**

### CODE REVIEW

File: src/it/scala/performanceanalysis/utils/TwitterFutures.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import com.twitter.util.{Return, Throw}
+import org.scalatest.concurrent.Futures
+
+/**
+  * Created by Ruud on 13-04-16.
+  */
+trait TwitterFutures extends Futures {
+
+  import scala.language.implicitConversions
```
Comment (TimSoethout) [inline src/it/scala/performanceanalysis/utils/TwitterFutures.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/scala-academy/performance-analysis/file/2403681944/issues/source?bid=3220915&fileBranchId=3232722#l11)

Comment (ruud) [inline src/it/scala/performanceanalysis/utils/TwitterFutures.scala]: Thanks Tim, but this does not resolve the issue. I still get the timeout when running the test.

Comment (ruud) [inline src/it/scala/performanceanalysis/utils/TwitterFutures.scala]: Add timeout also. Then it works!

**Comment by ruud** [comment]
Ready to be merged. Closes #11

**Pull request merged**